craft_providers/actions/snap_installer.py

```diff
@@ -189,8 +263,14 @@ def install_from_store(
             check=True,
             capture_output=True,
         )
+
```

Comment:
here and above, it would be great if you get rid of this change (it's a blank think, it just dirties diffs)